app/app.jsx

```diff
@@ -1,28 +1,22 @@
 import React from 'react';
+import { connect } from 'react-redux'
 import './scss/style.scss';
 import 'normalize.css';
 
 /* import components */
 import Main from './containers/main.jsx';
 import Nav from './containers/nav.jsx';
 
-class App extends React.Component {
+let App = () => {
 
-   render() {
-
-     const store = this.props.store;
-
-      return (
-         <div className="page">
-           <Nav
-             store={ store.getState() }
-           />
-           <Main
-             store={ store.getState() }
-           />
-         </div>
-      );
-   }
+  return (
```

Comment:
If the app is connected it will get the state as props, then it will need to pass them on to its children